src/kubernetes-ingress-controller/references/prometheus.md

```diff
@@ -5,21 +5,46 @@ title: Prometheus metrics
-* {{site.base_gateway}} can expose Prometheus metrics for served requests, if the [Prometheus plugin][prom-plugin] is enabled. See the [Using KongPlugin resource guide][kongplugin-guide] for information on how to enable a plugin. Also, we provide a specific [guide for integration with Prometheus and Grafana][grafana-guide] as well.
+* {{site.base_gateway}} can expose Prometheus metrics for served requests, if the [Prometheus plugin][prom-plugin] is enabled. See the [Using KongPlugin resource guide][kongplugin-guide] for information about how to enable a plugin. Also, we provide a specific [guide for integration with Prometheus and Grafana][grafana-guide] as well.
 
 This document is a reference for the former type.
 
-| Metric name | Description |
-|-------------|-------------|
-| `ingress_controller_configuration_push_count[success=true|false][protocol=db-less|deck]` | Count of successful or failed configuration pushes to Kong. <br><br> `protocol` describes the configuration protocol in use, which can be `db-less` or `deck`. <br><br> `success` logs the status of configuration updates. If `success` is `false`, an unrecoverable error occurred.  If `success` is `true`, the push succeeded with no errors.  |
-| `ingress_controller_translation_count[success=true|false]` | Count of translations from Kubernetes state to Kong state. <br><br> `success` logs the status of configuration updates. If `success` is `false`, an unrecoverable error occurred.  If `success` is `true`, the translation succeeded with no errors. |
-| `ingress_controller_configuration_push_duration_milliseconds[success=true|false][protocol=db-less|deck]` | The amount of time, in milliseconds, that it takes to push the configuration to Kong. <br><br> `protocol` describes the configuration protocol in use, which can be `db-less` or `deck`. <br><br> `success` logs the status of configuration updates. If `success` is `false`, an unrecoverable error occurred.  If `success` is `true`, the push succeeded with no errors. |
+## Prometheus metrics for configuration updates
 
-In addition to the above, {{site.kic_product_name}} exposes more low-level performance metrics - these may however change from version to version, as they are provided by underlying frameworks of {{site.kic_product_name}}.
+### ingress_controller_configuration_push_count
 
-A non-exhaustive list of these low-level metrics is present in the following places:
-* [controller-runtime metrics definition](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/internal/controller/metrics/metrics.go)
-* [workqueue metrics definition](https://github.com/kubernetes/component-base/blob/release-1.20/metrics/prometheus/workqueue/metrics.go#L29)
+`ingress_controller_configuration_push_count` provides the number of successful or failed configuration pushes to {{site.base_gateway}}.
+
+This metric expects that the following parameters are configured: 
```

Comment:
@czeslavo I added this to each metric section because the full metric example (ex. `ingress_controller_configuration_push_count[success=true|false][protocol=db-less|deck]`) was too long for the page. Is this accurate to call these "parameters"? And would customers know where to put these values without the example (ex. `ingress_controller_configuration_push_count[success=true|false][protocol=db-less|deck]`)?

Reply:
These parameters are called metric's labels. I reworded the sentences to specifically use this term ([b317f2a](https://github.com/Kong/docs.konghq.com/pull/4500/commits/b317f2ac099c08ff67455158362b301b095281ae)). They are populated by us while reporting the metric, and the user can inspect them. I think that it's going to be enough to have all possible values of every label explained in the way you've structured that. 👍